src/app/ApplicationTemplate.UWP/Views/Styles/Controls/TextBlock.xaml

```diff
@@ -49,7 +49,7 @@
 		<Setter Property="FontWeight"
 				Value="Bold" />
 		<Setter Property="Foreground"
-				Value="{StaticResource MaterialPrimaryDarkBrush}" />
```

Comment:
Topic for another PR but why aren,t we using MaterialHeadline1 and all ? Why are they duplicated here?

https://github.com/unoplatform/Uno.Themes#styles-for-basic-controls